apps/userdashboard/templates/a4_candy_userdashboard/userdashboard_overview.html

```diff
@@ -24,9 +24,9 @@ <h2 class="mt-sm-0 d-inline-block">{% trans 'Following' %}</h2>
 {% endif %}
 
 {# Translators: kosmo #}
-<h3>{% trans 'Activities' %}</h3>
+<h2>{% trans 'Activity' %}</h2>
```

Comment:
@fuzzylogic2000 so my understanding was that the public activities of the user should be 'Activities' and the private activity of other users on content should be 'Activity' because that's activity on their content, does that make sense?

Reply:
But then it should be fine like this! Now its says activity in the userdashboard (for both short and extended view) and activities on the profile.